### ISSUE TITLE
per-process files for draw and plot

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#232]
Imported from SourceForge on 2024-07-02 13:28:41
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2015-05-29 17:43:35
Original: https://sourceforge.net/p/maxima/patches/64

---

I did try to make a long-running maxima task faster by starting several processes at once but did get garbled plots when trying that: All maxima processes used the same temporary files.

This patch makes every maxima instance use different file names. 

Attachments:
- https://sourceforge.net/p/maxima/patches/64/attachment/0001-Ensure-per-process-filenames-for-concurrent-maxima.patch
- https://sourceforge.net/p/maxima/patches/64/attachment/0002-Documentation-of-the-new-filenames.patch

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-02 13:28:42
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2015-05-29 19:31:55
Original: https://sourceforge.net/p/maxima/patches/64/#e59e

---

- Attachments has changed:

Diff:

~~~~

--- old
+++ new
@@ -1 +1,2 @@
+0002-Documentation-of-the-new-filenames.patch (7.5 kB; text/x-diff)
 0001-Ensure-per-process-filenames-for-concurrent-maxima.patch (3.1 kB; text/x-diff)

~~~~

- **Group**:  -->

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-02 13:28:45
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2015-05-29 19:31:55
Original: https://sourceforge.net/p/maxima/patches/64/#3b06

---

Did add the necessary documentation updates as a second patch.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-02 13:28:47
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2015-10-28 05:54:55
Original: https://sourceforge.net/p/maxima/patches/64/#cbf0

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-02 13:28:49
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2015-10-28 05:54:55
Original: https://sourceforge.net/p/maxima/patches/64/#ec49

---

Resolved with commit f4af2a10e6471d1b9f9cd72c4aba887fa5c21621.
